tools/packaging/rpm/Dockerfile.build

```diff
@@ -1,14 +1,19 @@
 FROM centos:7
 
 RUN yum upgrade -y && \
+    yum remove git && \
+    yum install -y https://centos7.iuscommunity.org/ius-release.rpm && \
```

Comment:
I'm not liking iuscommunity.org being in the package provider list.

This could be an attack vector...